syntax = "proto3";

enum Status {
  Ok = 0;
  Error = 1;
  AlreadyConnected = 2;
}

message Hello {
  string id = 1;
}

message Bye {
  Status status = 1;
}

enum ActionType {
  Disconnect = 0;
  MoveUp = 1;
  MoveDown = 2;
  MoveRight = 3;
  MoveLeft = 4;
}
message Action {
  ActionType type = 1;
}
